### DISCUSSION TITLE
피드백 정리에 대해서...

### DISCUSSION THREAD

**Discussion by jin7969** [woowacourse-study/2022-code-review-study-2#24] [General]
피드백 정리를 하는 와중에..
순간적으로 숙제를 하는 느낌이 들었다.
그냥 빨리 하고 pr 날려야지 하는...
피드백을 1개 하더라도 내생각이 깊이 들어가서 하는것도 좋겠다 라는 생각을 하였고,
스터디 목적이 단순히 어떠한 정보를 알려주는 것도 있지만, 다양한 의견들로 같이 토론을 하여 생각을 넓혔으면 더 좋을것 같다.

이렇게 하자가 아니라 이러한 방법도 있으니 다양하게 시도해보면 좋치 않을까 생각 한다!
이부분도 같이 얘기 하고 싶다

**Comment by al-bur** [comment]
저는 `맡은 PR에 대한 리뷰중에 괜찮은 리뷰를 추려내는 작업`과 `깊게 분석해보고 싶은 PR은 자율적으로 분석을 해보는 작업`을 같이 하면 좋을 것 같습니다.

저는 리뷰어분들이 해주시는 리뷰들중에 비슷한 리뷰들이 있기 때문에 생각보다 리뷰를 정리하는 것은 부담스럽지 않았습니다만 다른 분들의 의견도 궁금하네요. (각자 PR 리뷰들만 정리하는데 얼마나 걸렸는지 이야기해보면 좋겠네요!)
